### ISSUE TITLE
pretrain problem

### ISSUE THREAD

**Issue by qian26** [ReinFlow/ReinFlow#11]
<img width="2559" height="1187" alt="Image" src="https://github.com/user-attachments/assets/66e9f732-b1fa-439c-a466-dbcf95927927" />

I followed the tutorial and trained a Rainflow strategy using `walker2d-medium-v2_pre_reflow_mlp_ta4_td20_seed42`, but the success rate was 0. Is this normal?

**Comment by qian26** [comment]
[W&B website](https://wandb.ai/2767213672-hnu/gym-walker2d-medium-v2-pretrain?nw=nwuser2767213672)
This is the W&B website for the pretrain.

**Comment by Tonghe-Zhang** [comment]
Have you tried running the evaluation script to test your policy after pretraining? @qian26

**Comment by Tonghe-Zhang** [comment]
Btw I noticed that the test frequency is 100 in walker config, since we only train 80 steps there were never an evaluation. You can try change it to 20 to see a fine grained logging of reward throughout the pretraining process. I think this should work. I already committed the change to [this yaml file](https://github.com/ReinFlow/ReinFlow/blob/release/cfg/gym/pretrain/walker2d-medium-v2/pre_reflow_mlp.yaml)

**Comment by jyan-R** [comment]
@Tonghe-Zhang Hi, thanks for releasing the code! I’m pretraining a diffusion policy following your instruction, and using the configuration exactly as in `cfg/gym/pretrain/walker2d-medium-v2/pre_diffusion_mlp.yaml`. I’m seeing the training loss plateau around ~0.19, and I’m wondering whether this is an expected range for this setup. Is a loss of this magnitude considered normal for the pretraining stage? Any publicly available Weights & Biases logs would be greatly appreciated 🌹

**Comment by Tonghe-Zhang** [comment]
@jyan-R Hi jyan, I think you are training a diffusion policy right? That part is borrowed from [DPPO](https://github.com/irom-princeton/dppo). Have you tested the policy with our [evaluation script](https://github.com/ReinFlow/ReinFlow/blob/release/agent/eval/eval_diffusion_agent.py)? That will be the most straightforward way of verification. Btw if you allow testing during training, it will also show you a periodic evaluation result. If it works and matches paper result I guess it will be fine. If it's not, I'd love to hear more details about your policy's reward, losses, etc. 

After some digging I found my pre-training curves obtained couple of months ago, and it looks like this:

<img width="1004" height="343" alt="Image" src="https://github.com/user-attachments/assets/1cb9af1e-0441-42ef-8bc6-0c68cc076be6" />

Although I haven't released them to our [wandb page](https://wandb.ai/reinflow/projects). My diffusion policy curves converges to 0.12 after couple of hundred steps. Will that help your research? Lemme know if you have any more questions.

**Comment by jyan-R** [comment]
That’s extremely helpful, thanks a lot! I'll go ahead for testing
